core/src/main/java/org/jdbi/v3/mapper/BuiltInMapperFactory.java

```diff
@@ -163,4 +169,15 @@ private static ZonedDateTime getZonedDateTime(ResultSet r, int i) throws SQLExce
         Timestamp ts = r.getTimestamp(i);
         return ts == null ? null : ZonedDateTime.ofInstant(ts.toInstant(), ZoneId.systemDefault());
     }
+
+    private static OffsetTime getOffsetTime(ResultSet r, int i) throws SQLException {
+        Time time = r.getTime(i);
+        ZoneOffset zoneOffset = Instant.now().atZone(ZoneId.systemDefault()).getOffset();
```

Comment:
This one is a problem: if somebody's system default zone observes daylight savings time, then offset will shift by an hour every time the zone offset shifts.

Example:

I live in the `United States/Denver` zone, which observes Daylight savings. Because I'm a rank amateur, I run my application server in local time instead of UTC.

During the winter, I store an `OffsetDateTime` of `15:10:05-07:00`. Because JDBC hasn't caught up with JSR-310, it gets saved to the database as just `15:10:05`.

Later, during the summer (after daylight savings has gone into effect), I query that row from the database, and get back `15:10:05`. The zone offset at this time is `-06:00`, so the result is `15:10:05-06:00`.

This makes me wonder whether JDBI could be a little opinionated on this one, and just store all time types in UTC.